walkthroughs/week-3-server/examples/todo-list/src/main/java/com/google/sps/servlets/ListTasksServlet.java

```diff
@@ -34,7 +34,9 @@
 @WebServlet("/list-tasks")
 public class ListTasksServlet extends HttpServlet {
 
-  @Override
+    private static final long serialVersionUID = 1L;
```

Comment:
In so far as Unique IDentifiers go, I suspect 1 isn't very unique.

Reply:
Yeah, that was the default option. I'll change it to a generated one.